packages/fast-form-generator-react/src/form/form-section.tsx

```diff
@@ -168,11 +169,42 @@ class FormSection extends React.Component<
         schemaLocation: string,
         dataLocation: string
     ): any => {
-        return this.props.location && this.props.location.onChange
+        this.props.location && this.props.location.onChange
             ? this.props.location.onChange(schemaLocation, dataLocation)
             : this.handleUpdateSection(schemaLocation, dataLocation);
     };
 
+    /**
+     * Generates a section link for properties
+     * that are objects
+     */
+    private generateFormItemSectionLink(
+        location: string,
+        property: any,
+        formItemProps: FormItemCommon
+    ): any {
```

Comment:
@janechu is this actually *any*? Looks like you're returning JSX?

Reply:
good catch